libraries/lib-music-information-retrieval/StftFrameProvider.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <vector>
+
+namespace MIR
+{
+class MirAudioReader;
+
+class StftFrameProvider
+{
+public:
+   StftFrameProvider(const MirAudioReader& source);
```

Comment:
Considering that crash I fixed in the unit test -- you might comment that the lifetime of the source encloses that of this?